charts/xui-mo-webapp/Chart.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: v1
 name: xui-mo-webapp
 home: https://github.com/hmcts/rpx-xui-manage-organisations
+<<<<<<< HEAD
+version: 0.1.384
+=======
 version: 0.2.0
+>>>>>>> develop
```

Comment:
Merge markers should be resolved

Reply:
this has just come in recently, ignore (conflicts come in)